www_src/pages/discover/mock.json

```diff
@@ -0,0 +1,126 @@
+[
+    {
+        "id": 1,
+        "version": "0.0.0",
+        "title": "Yam Fries",
+        "featured": true,
+        "history": {
+            "created_at": "2015-04-20T16:11:13.961Z",
+            "updated_at": "2015-04-20T16:11:13.961Z",
+            "deleted_at": null
+        },
+        "thumbnail": {
+            "480": "https://webmaker-app-mock.s3.amazonaws.com/tucker_000.jpg"
+        },
+        "author": {
+            "id": 1234,
+            "username": "billmurray",
+            "locale": {
```

Comment:
hmm, this feels like the API design might be too specific, in that case. Worth looking at there.